package.json

```diff
@@ -12,8 +16,8 @@
   "dependencies": {
     "bootstrap": "^3.4.0",
     "clipboard": "^2.0.8",
-    "jquery": "^3.6.0",
     "event-source-polyfill": "^1.0.25",
```

Comment:
Is event source polyfill something we won't need using babel? I recall babel to me a lot about transforming code to be compatible.

Reply:
We will definitely still need it, as it's not a *language* feature but a browser feature.